maintainers/maintainer-list.nix

```diff
@@ -16582,6 +16582,11 @@
       fingerprint = "1401 1B63 393D 16C1 AA9C  C521 8526 B757 4A53 6236";
     }];
   };
+  RoseHobgoblin = {
```

Comment:
make sure to lowercase this

Reply:
> make sure to lowercase this

is there a guideline that requires this?